dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/utils/OpenAPIUtils.java

```diff
@@ -13,11 +13,17 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
+import org.eclipse.microprofile.config.Config;
 import org.eclipse.microprofile.openapi.OASFactory;
 import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.eclipse.microprofile.openapi.models.info.Info;
 import org.eclipse.microprofile.openapi.models.servers.Server;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
Are you sure you want to use jackson directly and not the standard jsonp APIs?

Reply:
Sadly, I think so

1. We are already including Jackson as a dependency
2. We're calling `InfoReader.readInfo` from smallrye which takes a Jackson `ObjectNode`